src/platform/packages/shared/kbn-rrule/rrule.ts

```diff
@@ -112,12 +66,17 @@ export class RRule {
       .toDate();
 
     const nextRecurrences: Moment[] = [];
+    let iters = 0;
 
     while (
       (!count && !until) ||
       (count && yieldedRecurrenceCount < count) ||
       (until && current.getTime() < new Date(until).getTime())
```

Comment:
If `until` is already type `Date` why are we doing `new Date(until).getTime()`?

This is probably by mistake but at least it doesn't throw.

```
> const until = new Date('February 25, 2022 03:24:00')
undefined
> until
2022-02-25T02:24:00.000Z
> const foobar = new Date(until)
undefined
> foobar
2022-02-25T02:24:00.000Z
> foobar.getTime()
1645755840000
> until.getTime()
1645755840000
```

Reply:
agree, `until` is already a sanitized date, so it is safe to use until.getTime()